changelogs/unreleased/3497-no-redirect-start.yml

```diff
@@ -0,0 +1,6 @@
+description: The dashboard should no longer register as the start. This commit removes the redirect
+issue-nr: 3497
+change-type: patch
+destination-branches: [master]
+sections:
+  upgrade-note: "Clear your browser cache after upgrading"
```

Comment:
Some minor remarks:

* change-type: This is not really a bug fix, so the change type cannot be `patch`. But I am not sure whether we should call this a `minor` or a `major`. It's make it a `major` if we only merge this into `master`.
* destination-branches: At some point the idea was to merge this change into `master` and `iso4`, but from the ticket in the next column it looks like we abandoned that idea. @wouterdb Is that correct?
* upgrade-note: I would also mention why it could be required to clear the browser cache.

Reply:
master alone is OK